src/main/java/mil/dds/anet/resources/HomeResource.java

```diff
@@ -96,8 +97,8 @@ private String getBaseRequestUrl(URI requestlUrl) {
         // or using the default port of the scheme
         || HTTP_SCHEME.equals(scheme) && HTTP_DEFAULT_PORT == port
         || HTTPS_SCHEME.equals(scheme) && HTTPS_DEFAULT_PORT == port)
-            ? String.format("%s://%s/", scheme, host)
-            : String.format("%s://%s:%s/", scheme, host, port);
+            ? String.format("%s://%s/assets/client/logout.html", scheme, host)
```

Comment:
I would change it to something like:
```java
diff --git a/src/main/java/mil/dds/anet/resources/HomeResource.java b/src/main/java/mil/dds/anet/resources/HomeResource.java
index 5f6b3cfe2..85b16da1e 100644
--- a/src/main/java/mil/dds/anet/resources/HomeResource.java
+++ b/src/main/java/mil/dds/anet/resources/HomeResource.java
@@ -93,12 +93,13 @@ public class HomeResource {
     final String scheme = requestlUrl.getScheme();
     final int port = requestlUrl.getPort();
     final String host = requestlUrl.getHost();
+    final String logoutPage = "/assets/client/logout.html";
     return (port == -1 // no port in the request URL
         // or using the default port of the scheme
         || HTTP_SCHEME.equals(scheme) && HTTP_DEFAULT_PORT == port
         || HTTPS_SCHEME.equals(scheme) && HTTPS_DEFAULT_PORT == port)
-            ? String.format("%s://%s/assets/client/logout.html", scheme, host)
-            : String.format("%s://%s:%s/assets/client/logout.html", scheme, host, port);
+            ? String.format("%s://%s%s", scheme, host, logoutPage)
+            : String.format("%s://%s:%s%s", scheme, host, port, logoutPage);
   }
 
 }
```
and rename the method to `getLogoutUrl`.

Reply:
Would be nice to fix the typo in "requestl" as well :wink: